src/ui_basic/editbox.h

```diff
@@ -111,6 +111,10 @@ struct EditBox : public Panel {
 
 	ScopedTimer caret_timer_;
 	uint32_t caret_ms;
+	ScopedTimer cursor_movement_timer;
+	uint32_t cursor_ms;
+	bool cursor_movement_active = false;
```

Comment:
Ideally, all private variables should end with a trailing underscore. But since this is already mixed I'm leaving it up to you if you want to refactor this here or not.

Reply:
done. Could that be a codecheck? :)